src/Components/Components/src/RenderTree/RenderTreeEdit.cs

```diff
@@ -1,37 +1,47 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
+using System.Runtime.InteropServices;
 
 namespace Microsoft.AspNetCore.Components.RenderTree
 {
     /// <summary>
     /// Represents a single edit operation on a component's render tree.
     /// </summary>
+    [StructLayout(LayoutKind.Explicit)]
```

Comment:
Client-side Blazor has always relied on knowing the struct layout of `RenderTreeEdit`. This now makes it explicit that we're doing that and what offsets we expect.